index.html

```diff
@@ -351,6 +351,61 @@ <h4 class="featureText">Terminal</h4>
           </li>
         </ul>
       </section>
+      <section>
+        <ul class="flex-container" id="formContainer">
+          <li>
+            <p id="contactDescription">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+          </li>
+          <li>
+            <form action="https://formspree.io/f/xnqyndrw" method="POST">
+              <ul class="flex-container" id="formElements">
+                <li class="formElement">
+                  <input class="fElement" type="text" name="name" placeholder="Full name" maxlength="30" required>
+                </li>
+                <li class="formElement">
+                  <input type="email" name="email" placeholder="Email address" required>
+                </li>
+                <li class="formElement">
+                  <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+                </li>
+                <li id="submitForm">
+                  <button class="button" id="subBut" type="submit">Get in touch</button>
+                </li>
+              </ul>
+            </form>
```

Comment:
- [ ] You did an excellent job using `Formspree` to collect data! 🥇 I would only suggest that you style a bit the `<textarea>` so it matches the Figma design:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/213017945-bc36fed4-532a-40f0-b3e7-6e82665bea0b.png) | ![Screenshot from 2023-01-17 16-34-05](https://user-images.githubusercontent.com/77069011/213017877-07f2a513-d618-48b3-b0b4-0260ccbeb9ca.png) |


As you can see, your `<textarea>` is half the height of the Figma design, please fix it as well as the color of the social media icons! 🤓 🔧